1-syntax-basic-constructs/basic-js.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="robots" content="2">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <script src="js/basic.js"></script>
+    </head>
+<body>
+    <main>
+    <h1>Exercise Javascript</h1>  
+    <p>Click the button to demonstrate the prompt box.</p>
+    <button onclick="ValidAge()">Click.</button>
+    </main>
+</body>
+</html> 
```

Comment:
Get use to put script tags before closing the body tag always. 
```suggestion
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta name="robots" content="2">
        <meta name="viewport" content="width=device-width,initial-scale=1.0">
    </head>
<body>
    <main>
    <h1>Exercise Javascript</h1>  
    <p>Click the button to demonstrate the prompt box.</p>
    <button onclick="validateAge();">Click.</button>
    </main>
    <script src="js/basic.js"></script>
</body>
</html> 
```